geometry_msgs/Point32 position  # position in the real world, top-left of the patch
sensor_msgs/Image patch         # an image representing the patch of terrain to update